src/modules/tests/tests.selector.ts

```diff
@@ -75,3 +75,7 @@ export const isTestReportPracticeTest = (tests: TestsModel): boolean =>
 
 export const isEndToEndPracticeTest = (tests: TestsModel) : boolean =>
   startsWith(tests.currentTest.slotId, end2endPracticeSlotId);
+
+export const getAllTestStatuses = (tests: TestsModel): { [slotId: string]: TestStatus; } => {
```

Comment:
Will this work for old tests? The store is an in-memory repository, I was under the impression that we would need to access the local device storage in order to satisfy this user need?

Reply:
Do we have a means by which to test this against a representative scenario?

Reply:
The `testStatuses` property is storying the ones that we attempted to start or the ones that were in the devices local storage. Not all tests will have a status representation in this object.

Reply:
ah ok I can see it now, there is a `LoadPersistedTests` action that does it :)

Reply:
@roberthall12 can we rename the parameter to `test` as this selector operates on a single test at a time